CRM/PCP/Form/Campaign.php

```diff
@@ -64,8 +64,14 @@ public function preProcess() {
     parent::preProcess();
   }
 
+  /**
+   * Set default form values.
+   *
+   * @return array
+   *   Default values for the form.
+   */
   public function setDefaultValues() {
-    $dafaults = array();
```

Comment:
Funny thing is that php won't care. Unless of course the two conditionals below are false. I guess one or the other is always true or else people would have noticed an undefined var notice.